valkey/values.yaml

```diff
@@ -1,7 +1,15 @@
+global:
+  # the global image registry
```

Comment:
Capitalize "The". Can you add a comment that setting global registry will override the registry defined under `image`?